api/src/main/resources/liquibase-schema-only.xml

```diff
@@ -23,6 +23,6 @@
 		http://www.liquibase.org/xml/ns/dbchangelog 
 		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
 	
-	<include file="org/openmrs/liquibase/snapshots/schema-only/liquibase-schema-only-2.6.x.xml"/>
```

Comment:
Is there a reason why you changed this?

Reply:
I was trying to run get my changelog to run.

Reply:
Can you then revert the change?